.github/workflows/docker.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
-            ref: ${{ inputs.commit || 'main' }}
+            ref: ${{ github.ref || inputs.commit || 'main' }}
```

Comment:
github.ref will be the release tag when the workflow is triggered by github release.